USFMToolsSharp/HtmlRenderer.cs

```diff
@@ -8,10 +8,32 @@ namespace USFMToolsSharp
     public class HtmlRenderer
     {
         public List<string> UnrenderableTags;
+        public List<string> FootnoteTextTags;
 
+        private bool isSingleSpaced = true;
+        private bool hasOneColumn = true;
+        private bool isL2RDirection = true;
+        private bool isTextJustified = false;
+        private bool separateChapters = false;
+
+        private string InsertedHTML = "";

```

Comment:
A better name for this would be FrontMatterHtml because InsertedHTML isn't too descriptive as to what it is going to be used for.